lte/gateway/fabfile.py

```diff
@@ -484,6 +484,72 @@ def integ_test(
         env.hosts = [gateway_host]
 
 
+def integ_test_deb_installation(
```

Comment:
Imho this should not be duplicated - the existing `integ_test` function could be extended.

Reply:
I refactored both functions to avoid duplicate code.